piperider_cli/dbt/utils.py

```diff
@@ -185,7 +185,7 @@ def display_changes(self, b, t, format_text: str, *, converter: Callable = None,
         if negative_change:
             color = "red" if t > b else "green"
         else:
-            color = "green" if t > b else "red"
+            color = "orange" if t > b else "orange"
```

Comment:
not sure if "number of all tests" is a negative change or not?

Reply:
I put it as neutral because added or removed are human-modified results.